src/Php73/bootstrap.php

```diff
@@ -0,0 +1,18 @@
+<?php
+
+/*
+ * This file is part of the Symfony package.
+ *
+ * (c) Fabien Potencier <fabien@symfony.com>
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+if (PHP_VERSION_ID < 70300) {
+    if (!function_exists('is_countable')) {
+        function is_countable($s) {
```

Comment:
the function should be on one single line (to make the test suite work as expected)
`$s` should be renamed by `$var` I suppose also.